app/assets/javascripts/services/bridge.ts

```diff
@@ -1,24 +1,30 @@
 /** Platform-specific (i-e Electron/browser) behavior is handled by a Bridge object. */
 export interface Bridge {
-  getKeychainValue(): Promise<unknown>;
+  getKeychainValue(): Promise<any>;
   setKeychainValue(value: any): Promise<void>;
   clearKeychainValue(): Promise<void>;
 }
 
 const KEYCHAIN_STORAGE_KEY = 'keychain';
 
 export class BrowserBridge implements Bridge {
-  async getKeychainValue(): Promise<unknown> {
-    const value = localStorage.getItem(KEYCHAIN_STORAGE_KEY);
+
+  async getKeychainValue(): Promise<any> {
+    const value = localStorage.getItem(this.keychainStorageKey);
     if (value) {
       return JSON.parse(value);
     }
   }
+
   async setKeychainValue(value: any): Promise<void> {
-    localStorage.setItem(KEYCHAIN_STORAGE_KEY, JSON.stringify(value));
+    localStorage.setItem(this.keychainStorageKey, JSON.stringify(value));
   }
 
   async clearKeychainValue(): Promise<void> {
-    localStorage.removeItem(KEYCHAIN_STORAGE_KEY);
+    localStorage.removeItem(this.keychainStorageKey);
+  }
+
+  get keychainStorageKey() {
+    return KEYCHAIN_STORAGE_KEY;
   }
```

Comment:
The keychain storage key is a bridge-specific implementation detail that shouldn't be exposed to other parts of the code (see other comments for the code that relies on this property)